build.gradle

```diff
@@ -61,3 +61,13 @@ task release(type: Zip) {
     from jar
     into project.name
 }
+
+task copyToMods(type: Copy) {
+    if (!project.hasProperty('game.full_path')) {
+        throw new GradleException("Missing game.full_path in gradle.properties. See gradle.properties.dist as an example.")
```

Comment:
CI fails due to that. Perhaps just a warning (or error) log with this string and early return?